locales/zh_CN/LC_MESSAGES/install.po

```diff
@@ -45,60 +45,60 @@ msgstr "对于Arch Linux，请执行 ``yaourt -S ncurses5-compat-libs`` 安装
 
 #: ../../install.rst:17
 msgid ""
-"On Windows, please install `Microsoft Visual C++ Redistributable "
-"<https://aka.ms/vs/16/release/vc_redist.x64.exe>`_ if you haven't."
+"On Windows, please install `Microsoft Visual C++ Redistributable <https://"
+"aka.ms/vs/16/release/vc_redist.x64.exe>`_ if you haven't."
 msgstr ""
-"对于Windows，请预先安装运行组件库 `Microsoft Visual C++ Redistributable "
+"对于 Windows，请预先安装运行组件库 `Microsoft Visual C++ Redistributable "
 "<https://aka.ms/vs/16/release/vc_redist.x64.exe>`_ ."
 
 #: ../../install.rst:21
 msgid "Troubleshooting"
 msgstr "故障排除"
 
 #: ../../install.rst:24
-#, fuzzy
 msgid "Windows issues"
-msgstr "Linux相关问题"
+msgstr "Windows 相关问题"
 
 #: ../../install.rst:26
-#, fuzzy
 msgid ""
-"If Taichi crashes and reports ``ImportError`` on Windows: Please consider"
-" installing `Microsoft Visual C++ Redistributable "
-"<https://aka.ms/vs/16/release/vc_redist.x64.exe>`_."
+"If Taichi crashes and reports ``ImportError`` on Windows: Please consider "
+"installing `Microsoft Visual C++ Redistributable <https://aka.ms/vs/16/"
+"release/vc_redist.x64.exe>`_."
 msgstr ""
-"对于Windows，请预先安装运行组件库 `Microsoft Visual C++ Redistributable "
-"<https://aka.ms/vs/16/release/vc_redist.x64.exe>`_ ."
+"如果在 Windows下 Taichi 崩溃并报告 ``ImportError`` ：请考虑安装运行组件库 "
+"`Microsoft Visual C++ Redistributable <https://aka.ms/vs/16/release/"
+"vc_redist.x64.exe>`_ 以进行解决."
 
 #: ../../install.rst:29
-#, fuzzy
 msgid "Python issues"
-msgstr "OpenGL相关问题"
+msgstr "Python 相关问题"
 
 #: ../../install.rst:31
 msgid "If ``pip`` complains that it could not find a satisfying package, i.e.,"
-msgstr ""
+msgstr "如果 ``pip`` 向你抱怨找不到合适的包，即,"
 
 #: ../../install.rst:33
 msgid ""
-"ERROR: Could not find a version that satisfies the requirement taichi "
-"(from versions: none)\n"
+"ERROR: Could not find a version that satisfies the requirement taichi (from "
+"versions: none)\n"
 "ERROR: No matching distribution found for taichi"
```

Comment:
Not translated?

Reply:
The error message of the terminal is in English. I think keeping the original text may help readers relate to their own situation better 🧐 What's your opinion ,boss？

Reply:
Then follow your decision. :+1: